### DISCUSSION TITLE
accessing-android-app-secret-from-github-actions-using-gradle/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#10] [Comments]
# accessing-android-app-secret-from-github-actions-using-gradle/

Often an open source project will have API keys, auth tokens, and other secrets that definitely shouldn’t end up in source code. In my current open source project (a rewrite of APOD Wallpaper) I needed to store my APOD API key.

https://blog.jakelee.co.uk/accessing-android-app-secret-from-github-actions-using-gradle/

**Comment by morganesoula** [comment]
Hey there, thank you very much for this! It was really really helpful even after two years :)
